vm/src/builtins/mod.rs

```diff
@@ -29,6 +29,7 @@ pub use function::PyFunction;
 pub(crate) mod generator;
 pub use generator::PyGenerator;
 pub(crate) mod getset;
+pub use getset::PyGetSet;
```

Comment:
oh, i thought this is internally used only but not in this case. maybe we'd better to expose builtinfunc also.